server/routes/server.js

```diff
@@ -11,8 +11,20 @@ function convertToClientFormat(selected_config, esResponse) {
       var flatten = require('flat');
       source = flatten(source);
     }
+    
+    var display = source[selected_config.fields.mapping['display_timestamp']];
+    
+    if (selected_config.format_timestamp != null){
+      var moment = require('moment-timezone');
+      display = moment(source[selected_config.fields.mapping['timestamp']]);
```

Comment:
Shall we use display_timestamp field here. If there are no ES fields suitable for direct display, use can map @timestamp to display_timestamp.